### PR TITLE
Add general settings and wire up turn settings to facilitate testing turn servers.

### PR DESCRIPTION
**Pull request by andresusanopinto**

### CODE REVIEW

File: src/ui/testrtc-main.html [hunk 1]
```diff
@@ -48,18 +48,65 @@
...
+        <div class="select"><label>Video source: <select id="videoSource"></select></label></div>
+      </div>
+
+      <div>
+        <h3>TURN</h3>
+        <paper-input label="URI" always-float-label placeholder="E.g.: turn:myserver.com:3478" value="{{settings.turnURI}}"></paper-input>
+        <paper-input label="Username" always-float-label value="{{settings.turnUsername}}"></paper-input>
+        <paper-input label="Credential" always-float-label value="{{settings.turnCredential}}"></paper-input>
+      </div>
+      <p>For convenience here is a link with this settings:</p>
```
Comment (KaptenJansson) [inline src/ui/testrtc-main.html]: this settings > these settings

File: src/ui/testrtc-main.html [hunk 2]
```diff
@@ -48,18 +48,65 @@
...
 
     <!-- Bug report dialog -->
     <report-dialog id="bugreport"></report-dialog>
   </template>
   <script>
+    // Parse URL parameters and configure test filters.
+    function parseUrlParameters() {
+      var output = {};
+      if (window.location.search !== '') {
+        // python SimpleHTTPServer always adds a / on the end of the request.
```
Comment (KaptenJansson) [inline src/ui/testrtc-main.html]: Can you really run testrtc using SimpleHTTPServer anymore? ;)

File: src/ui/testrtc-main.html [hunk 3]
```diff
@@ -48,18 +48,65 @@
...
+        <h3>Media</h3>
+        <div class="select"><label>Audio source: <select id="audioSource"></select></label></div>
+        <div class="select"><label>Video source: <select id="videoSource"></select></label></div>
+      </div>
+
+      <div>
+        <h3>TURN</h3>
+        <paper-input label="URI" always-float-label placeholder="E.g.: turn:myserver.com:3478" value="{{settings.turnURI}}"></paper-input>
+        <paper-input label="Username" always-float-label value="{{settings.turnUsername}}"></paper-input>
+        <paper-input label="Credential" always-float-label value="{{settings.turnCredential}}"></paper-input>
```
Comment (KaptenJansson) [inline src/ui/testrtc-main.html]: Credential > Password

File: src/ui/testrtc-main.html (hunk 2 above)
Comment (andresusanopinto) [inline src/ui/testrtc-main.html]: Yes. I prefer polyserve myself.

File: src/ui/testrtc-main.html (hunk 3 above)
Comment (andresusanopinto) [inline src/ui/testrtc-main.html]: PeerConnection refers to that as a credential (not password). So I prefer to use the same terminology.

File: src/ui/testrtc-main.html (hunk 1 above)
Comment (andresusanopinto) [inline src/ui/testrtc-main.html]: Done.

**Comment by KaptenJansson** [comment]
LGTM

**Pull request merged**